modules/programs/direnv.nix

```diff
@@ -143,9 +143,7 @@ in {
                 let direnv = (
                     # We want to get the stdout from direnv even if it exits with non-zero,
                     # because it will have the DIRENV_ internal variables defined.
-                    do { ${getExe cfg.package} export json }
-                    | complete
-                    | get stdout
+                    do --ignore-program-errors { ${getExe cfg.package} export json }
```

Comment:
Looks OK to me but CI is complaining about formatting:

    /home/runner/work/home-manager/home-manager/modules/programs/direnv.nix: not formatted